### ISSUE TITLE
"http framework are on the way" in roadmap, is this independent of cloudwego/volo or fuse with volo?

### ISSUE THREAD

**Issue by ouvaa** [monoio-rs/monoio#253]
"http framework are on the way" in roadmap, is this independent of cloudwego/volo or fuse with volo?

when can we expect this feature to be out? possible to provide an eta?

**Comment by ihciah** [comment]
No. There's no any relationship between monoio and volo for now.
We are building a http component here: https://github.com/monoio-rs/monoio-http

**Comment by ouvaa** [comment]
@ihciah possible to provide eta on when framework will be ready?
sorry to trouble but i have no idea how to make h2 work with tls 
https://github.com/monoio-rs/monoio-http/blob/master/monoio-http/examples/h2_monoio_server.rs
as it is, it's not "usable".

cannot display on browser. is it because the tls part is terminated by ktls or something? no idea why it's purposefully left out.

appreciate if u can put an eta on this. looking forward to the framework

**Comment by ouvaa** [comment]
@ihciah pls look at https://salvo.rs/book/

it's the simplest framework for developer use. hope monoio http framework can be this easy too.
also, http3 is necessary, please consider it as part of the roadmap. thx
